README.md

```diff
@@ -5,26 +5,66 @@ Workspace is a tool to orchestrate and bring consistency to your project environ
 ## Documentation
 
 ### Getting Started
+
 #### Requirements
+
  - `PHP-7.2+`
  - `sodium` php extension installed and activated in php.ini if it's not enabled by default 
  - `docker 17.04.0+`
  - `docker-compose (compose file version 3.1+)`
+
 #### Installation
-Download the `ws` file from the [Latest Release](https://github.com/my127/workspace/releases/latest) make executable and move to a location in your PATH, eg.
+
+Download the `ws` file from the [Latest Release](https://github.com/my127/workspace/releases/latest) make executable and 
+move to a location in your PATH, eg.
+
 ```
 chmod +x ws && sudo mv ws /usr/local/bin/ws
 ```
-#### Creating a workspace
-```
-# TODO
-```
-### Anatomy of a workspace
 
 #### Key Concepts
- - [Workspace](docs/concepts/workspace.md)
- - [Harness](docs/concepts/harness.md)
+
+##### Workspace
+ 
+`workspace` describes both the CLI tool `ws` and the project environment itself. It is dependent on a harness definition.
+ 
+##### Harness
  
+ A harness is a "flavour" of workspace which defines the blueprint to which a project environment will adhere. It  
+ incorporates two distinct elements:
+ 
+ 1. *Skeleton* - a project stub applied when the workspace is first created with a given harness 
+ 2. *Overlay* - concrete files that are always applied to the workspace
+ 
+ We currently have harnesses to support the following frameworks: 
+ 
+ * Magento 1 & 2
+ * Drupal 8
+ * Spryker
+ * Akeneo
+ * Node
+ * Wordpress
+ 
+## Commands
+
+By default a workspace project ships with the following commands:
+
+| Command                                    | Description                                             |
+|---                                         |---                                                      |
+|`ws install`                                |Install project                                          |
+|`ws enable`                                 |Start a previously disabled environment                  |
```

Comment:
This project only defines:
```
ws

Sub Commands:
   global       
   secret       
   install      
   harness      

Global Options:
  -h, --help    Show help message
```
```
ws global service <name> <action> [options]
ws global config get <key>
ws secret encrypt <message> [<key>] [options]
ws secret decrypt <encrypted> [options]
ws secret generate-random-key

ws install

Usage:
  ws install [options]


Command Options:
      --step=<step>    Step from which to start installer.
      --skip-events    If set events will not be triggered.

ws harness download
ws harness prepare
```
The rest are from harnesses, and may not exist.

Reply:
Is it fair to say most harnesses implement these universal commands?

Reply:
At this point i'd suggest using `ws` without a harness is the edge case